src/components/accordion/Accordion.stories.js

```diff
@@ -40,26 +40,38 @@ export const Accordion = {
         collapseLabel="${args.collapseLabel}"
       >
         <kd-accordion-item opened @on-toggle=${(e) => action(e.type)(e)}>
-          <span slot="title">Accordion Title 1</span>
-          <span slot="subtitle">Accordion subtitle 1</span>
+          <span slot="title" aria-label="Accordion Title 1"
```

Comment:
I don't think any of these aria-labels are necessary since it duplicates the visible text.

Reply:
but if we didn't use aria-label then screen readers wont read **title** and **subtitle** of accordion

Reply:
Are you saying let application handle this ?

Reply:
But why won't they read it? The text is present and visible.

Reply:
Because of slotted elements. Anyways you reverted this and converted this to textString then now it ll readable.